src/controllers/home.js

```diff
@@ -1,3 +1,8 @@
+const text = require('../text.js');
+
 module.exports = (req, res) => {
-  res.render('home');
+  res.render('home', {
+    english: text.en.english,
```

Comment:
can use `req.app.locals.en.english` instead, since we set this in `app.js`, then there's no need to require `text.js`

https://github.com/foundersandcoders/OTP-Data-Entry/blob/7f7c26f6d83b51fbcf7882e49f8f0d0d2d8793cb/src/app.js#L13-L14